domains/authorization/Authorization.API/API/Models/ApplicationDbContext.cs

```diff
@@ -26,23 +26,15 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
     private static void ConfigureOrganizationTable(ModelBuilder modelBuilder)
     {
-        modelBuilder.Entity<Organization>().Property(o => o.OrganizationName)
+        modelBuilder.Entity<Organization>().Property(o => o.Name)
             .HasConversion(new ValueConverter<OrganizationName, string>(v => v.Value, v => new OrganizationName(v)))
             .IsRequired();
 
         modelBuilder.Entity<Organization>().Property(o => o.Tin)
             .HasConversion(new ValueConverter<Tin, string>(v => v.Value, v => new Tin(v)))
             .IsRequired();
 
-        modelBuilder.Entity<Organization>().Property(o => o.IdpId)
-            .HasConversion(new ValueConverter<IdpId, Guid>(v => v.Value, v => new IdpId(v)))
-            .IsRequired();
-
-        modelBuilder.Entity<Organization>().Property(o => o.IdpOrganizationId)
-            .HasConversion(new ValueConverter<IdpOrganizationId, Guid>(v => v.Value, v => new IdpOrganizationId(v)))
-            .IsRequired();
-
-        modelBuilder.Entity<Organization>().HasIndex(o => o.IdpOrganizationId).IsUnique();
+        modelBuilder.Entity<Organization>().HasMany(it => it.Affiliations);
```

Comment:
Why the change here?

Reply:
To generate the sql so an organization as many affiliations.